languages/python/flask-oso/tests/test_flask_oso.py

```diff
@@ -55,6 +55,8 @@ def test_initialization(flask_app, oso, simple_policy, app_ctx, user):
     flask_oso.set_oso(oso)
     flask_oso.authorize(action="read", resource="resource")
 
+
+def test_initialization_with_init(flask_app, oso, simple_policy, app_ctx, user):
```

Comment:
@dhatch I had to split these tests, since initializing FlaskOso twice with the same Oso instance attempts to register Request twice, which now (correctly) raises an exception.